tools/pre_ci.ps1

```
@@ -0,0 +1,60 @@
+# Group Balancer: Local Pre-CI Script
+# Automates deterministic checks: Linting, Dead Code, Docstrings, Tests, README, and Build.
+
+$ErrorActionPreference = "Stop"
+
+function Check-ExitCode {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Check-ExitCode' uses an unapproved verb.

(PSUseApprovedVerbs)